radix-engine-macros/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 [dependencies]
 proc-macro2 = "1.0.66"
 quote = "1.0.33"
-syn = { version = "2.0.29", features = ["full"] }
+syn = { git = "https://github.com/dtolnay/syn.git", tag = "1.0.93", features = ["full", "extra-traits"] }
```

Comment:
Can we use v1 everywhere? Or just v2? (Maybe we should just update to v2 everywhere? Although was it v2 which was hitting issues on Cargo?)

Reply:
I don't really like linking to a tag on an external repo (in case they move the tag) - perhaps we should create a fork? Or maybe the lock file protects us

Reply:
> Can we use v1 everywhere?

After this PR, we use v1 everywhere. I'd like to upgrade to v2 everywhere in near future.

> I don't really like linking to a tag on an external repo (in case they move the tag) - perhaps we should create a fork? Or maybe the lock file protects us

I'm cool either way, and yes, Node cargo.lock does protects us I figure.